Cargo.toml

```diff
@@ -85,7 +89,11 @@ secp256k1 = ["libp2p-core/secp256k1"]
 rsa = ["libp2p-core/rsa"]
 ecdsa = ["libp2p-core/ecdsa"]
 serde = ["libp2p-core/serde", "libp2p-kad?/serde", "libp2p-gossipsub?/serde"]
-tokio = ["libp2p-mdns?/tokio", "libp2p-tcp?/tokio", "libp2p-dns?/tokio"]
+onion-async-std = ["dep:libp2p-onion", "libp2p-onion?/async-std"]
+onion-tokio = ["dep:libp2p-onion", "libp2p-onion?/tokio"]
```

Comment:
Please remove these in favor of a single `onion` one and forwarding of the tokio and async-std ones in the `tokio` and `async-std` feature.